libs/checkpoint-postgres/langgraph/checkpoint/postgres/base.py

```diff
@@ -224,7 +224,7 @@ def _dump_writes(
         ]
 
     def _load_metadata(self, metadata: dict[str, Any]) -> dict[str, Any]:
-        return self.jsonplus_serde.loads(self.jsonplus_serde.dumps(metadata))
```

Comment:
I think this change also needs to be undone?

Reply:
yup, needs to be undone